src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/User.orm.xml

```diff
@@ -49,10 +49,10 @@
             <cascade>
                 <cascade-all/>
             </cascade>
-            <join-column name="customer_id" referenced-column-name="id" nullable="false" />
+            <join-column name="customer_id" referenced-column-name="id" nullable="false" on-delete="CASCADE" />
```

Comment:
But when I want to create a new user for that customer I'm getting aduplicate email error! How should I create a user for an existing customer?